imports/components/controls.jsx

```diff
@@ -0,0 +1,204 @@
+import { DropdownButton, MenuItem, Modal, Button } from 'react-bootstrap';
+import { PlayerList } from './player_list';
+
+class RevealControl extends React.Component {
```

Comment:
...eww, really? Ugh that seems like a terrible decision. Even with the library, I'll note that IMO the biggest drawback about the ES6 class style is that you can't declare proptypes at the top of your classes.